### DISCUSSION TITLE
Clojure support

### DISCUSSION THREAD

**Discussion by hballangan-mdsol** [open-telemetry/opentelemetry-java-instrumentation#2330] [Q&A] [answered]
Hi, is Clojure supported by the opentelemetry java agent? If not, will there be any plans to support it?

**Comment by iNikem** [comment]
We don't currently have any specific support for Clojure libraries. PRs are welcomed! :) In theory it does not differ from any other JVM-based language that we already support, like Kotlin or Scala.

**Comment by steffan-westcott** [comment, marked as answer]
I recently released a library [`clj-otel`](https://github.com/steffan-westcott/clj-otel) that supports manual enrichment of telemetry produced by the OpenTelemetry Java agent using idiomatic Clojure. The [tutorial](https://github.com/steffan-westcott/clj-otel/blob/master/doc/tutorial.adoc) is a good example of manually adding spans and attributes to automatic instrumentation.

**Reply by hballangan-mdsol** [reply]
Late reply but thanks for sharing the library! Looks like it's sending trace as expected :D
